### ISSUE TITLE
Not possible to unmap Space button to be used for other action than Play/Pause

### ISSUE THREAD

**Issue by rRobis** [SubtitleEdit/subtitleedit#8424]
It seems that currently, it's not possible to use Space for anything else in the waveform section, because it seems that it's "baked" in the code to video play/pause.

Toggle play/pause shortcut should be used for this, and user should be able to remap it to whatever button he wants, and make space do something else.

**Comment by lambdacore12** [comment]
@rRobis What do you want to do with space?

**Comment by rRobis** [comment]
Unmap it or use it for something else. I use space as a modifier in ahk for subtitle edit, and sometimes for some reason it bugs out and bypasses ahk shortcut and presses space and starts playing the video.

**Comment by lambdacore12** [comment]
Yeah, I wanted to suggest AHK to fix your problem, but since you already use it...
Can you share your AHK script so we can think for a solution?

**Comment by rRobis** [comment]
```
#HotIf winActive("ahk_exe SubtitleEdit.exe") and !CaretGetPos() 
; --- Mouse
t & WheelUp::SendInput("{F23}")  ; Move text up
t & WheelDown::SendInput("{F22}") ; Move text down

Space & WheelUp::SendInput("{F21}") ; Move text between next subtitle
Space & WheelDown::SendInput("{F20}")
; ----
```

I used to have direct remap instead of SendInput, but I think that now it's more stable. 
I don't remember, if it fired off play with this method or not, i have to test more when i have more tasks in subtitle edit.
........................


In other place i also have this to allow to press space in other dialogue windows.
```
#HotIf winActive("ahk_exe SubtitleEdit.exe") and !winActive("Subtitle Edit")
Space::SendInput("{Space}")
#HotIf
```

**Comment by lambdacore12** [comment]
I see. I'll do some test of my own as well.

**Comment by rRobis** [comment]
It doesn't always make it play, but there are some circumstances when it fires play with this. I can't remember now what exactly I did to make it happen.
